athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandler.java

```diff
@@ -158,16 +157,20 @@ protected void readWithConstraint(final BlockSpiller spiller, final ReadRecordsR
                         final Map<Object, Object> obj = graphTraversalFinal.next();
                         return (rowWriter.writeRow(block, rowNum, (Object) obj) ? 1 : 0);
                     });
-                } catch (final Exception e) {
+                } 
+                catch (final Exception e) {
```

Comment:
Catching Exception is a bad practice, please catch only RuntimeException + any specific checked exceptions that the code requires (for example IOException is a specific exception that the compiler will complain about if code in your 'try' can throw it but you dont have a corresponding catch.

Reply:
Fixed.